### DISCUSSION TITLE
What use case is survey.geometry used for?

### DISCUSSION THREAD

**Discussion by rbroth** [micronutrientsupport/database-architecture#192] [General]
We're storing geometry for  the area covered by surveys, but do we need to? Can't we just link to a "country" table?

Arguments for doing it the way we're doing it now:
- Countries breaking apart e.g. Sudan
- Surveys that cover multiple countries

Against:
- It takes up lots of space

@bgsandan What is the actual use case that the data is used for? It's not aggregation, that's a different table. Will users care about "historical" surveys?

**Comment by bgsandan** [comment]
Pretty much the arguments for, plus potentially surveys that cover a smaller geographic area than a country.

Storage is cheap and we won't have that many surveys!

**Reply by rbroth** [reply]
What are we actually using the data for though? Is it for displaying the available datasets on a map for the user to select? For somehow aggregating data from multiple surveys? Something else?

**Reply by bgsandan** [reply]
Primarily for spatial query of which datasets intersect with aggregation location of interest (which could be a country). Quite possible that they will also be displayed on a map at some point.  These are essentially an example of data of type 1 from the slide the other day
